courses/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+
+# Create your models here.
+
+class Languages(models.Model):
+    language_name = models.CharField(max_length=20)
+
+    def __str__(self):
+        return self.language_name
+
+
+class Domains(models.Model):
+    domain_name = models.CharField(max_length=20)
+
+    def __str__(self):
+        return self.domain_name
+
+class KnowledgeBase(models.Model):
+    SKILL_BEGINNER = 'beg'
```

Comment:
Two scoops (TSOD) says capital letters with a limit of 2 chars eg. `BG`. Also, move them to a constants file. inside the app create a utils package and then create a constants file for storing the constants used in the app (Another TSOD tip)